c2pa/c2pa_api/c2pa_api.py

```diff
@@ -39,7 +39,7 @@ class Reader(api.Reader):
     def __init__(self, format, stream, manifest_data=None):
         super().__init__()
         if manifest_data is not None:
-            self.from_manifest_data_and_stream(format, manifest_data, stream)
+            self.from_manifest_data_and_stream(manifest_data, format, stream)
```

Comment:
Shouldn't the `stream` here also be wrapped in a `C2paStream`?

Reply:
The unit test modified in this PR will fail with a hash mismatch if the stream arg (`output` var in test) is not wrapped in `C2paStream` ahead of time when `validation_status` is checked:

```python
self.assertNotIn("validation_status", reader.json())
```

Wrapping the stream arg before calling init makes the test pass but it seems like it should be done in the `Reader` init
![image](https://github.com/user-attachments/assets/3d1e6f0a-e8ab-4111-a407-bf8804c2187b)

Reply:
Filed a [PR](https://github.com/contentauth/c2pa-python/pull/48) for this so it didn't get lost